src/modules/emote_socket_listeners/youtube.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import socketClient, {EventNames} from '../../socket-client.js';
+import watcher from '../../watcher.js';
+import channelEmotes from '../emotes/channel-emotes.js';
+import YoutubeHiddenMessage from '../../common/components/YoutubeEphemeralMessage.jsx';
+import {getLiveChat} from '../../utils/youtube.js';
+import {validChannelDestination} from './common.js';
+
+async function sendEphemeralMessage(message) {
```

Comment:
this code is almost identical for each platform. is there a way to make only the chat notice be platform-specific?